apps/fabric-website/src/utilities/svgIcons.tsx

```diff
@@ -215,7 +214,7 @@ export const WebLogo = (props?: ISVGIconProps) => {
 
 export const CrossPlatformLogo = (props?: ISVGIconProps) => {
   props = props || {};
-  const id = useId('crossplat');
+  const id = getId('crossplat');
```

Comment:
I'm not sure that makes sense. It make more sense to do any of the following:

1. Change them to be used as components. E.g. change the register process to render them:
```tsx
registerIcons({
  icons: {
     CrossPlatformLogo: <CrossPlatformLogo />
  }
});
```

2. If they truely are just functions, they should be renamed and not capitalized.

Reply:
yeah, i changed them to non caps. turns out we don't need that ID on the offending items (we dont' use the color version of the icon anymore). The homepage icons (color) are different icons and are only used once.